modules/ROOT/pages/platform/security/secure-connections.adoc

```diff
@@ -65,7 +93,7 @@ Aura VPCs can't initiate connections back to your VPCs.
 ==== Browser and Bloom access over private endpoints
 
 To connect to your instance via Browser or Bloom, you must use a dedicated VPN.
-This is because when you disable public access to your instance, this applies to all connections, including those from your computer when using Browser or Bloom.
+This is because when you xref:[disable public access] to your instance, this applies to all connections, including those from your computer when using Browser or Bloom.
```

Comment:
Do you think I should link to the disabling public access section here?

Reply:
Yes, why not? I think it would be helpful.

Reply:
You'll need to insert the xref here.